Reply/app/src/main/java/com/materialstudies/reply/ui/compose/ComposeFragment.kt

```diff
@@ -69,13 +89,24 @@ class ComposeFragment : Fragment() {
             email = composeEmail
 
             composeEmail.nonUserAccountRecipients.forEach { addRecipientChip(it) }
+            composeEmail.nonUserAccountRecipients.forEach{  addRecipientsCard(it) }
+
+            binding.handlers = containerTranformHandler
+
+            binding.recipientAddIcon.setOnClickListener {
+                containerTransformRecipientChipToCard()
```

Comment:
should the add icon be doing this? i thought the add icon would be responsible for adding a new chip, which we don't really need for the demo purposes